R/layer_residual_quantiles.R

```diff
@@ -0,0 +1,88 @@
+#' Creates predictions based on residual quantiles
+#'
+#' @param frosting a `frosting` postprocessor
+#' @param ... Unused, include for consistency with other layers.
+#' @param probs numeric vector of probabilities with values in (0,1)
+#'   referring to the desired quantile.
+#' @param symmetrize logical. If `TRUE` then interval will be symmetric.
+#' @param name character. The name for the output column.
+#' @param .flag a logical to determine if the layer is added. Passed on to
```

Comment:
Is `.flag` still preserved or did we agree to remove it?

Reply:
It's there for now, just for consistency with the other `layer_*()` functions. But the plan is to change the name later. (Separate PR).